stdlib/runtime-api/src/main/ballerina/src/runtime/errors.bal

```diff
@@ -29,9 +29,9 @@ public type CallStackElement record {|
     int lineNumber;
 |};
 
-# Retrieves the Call Stack
+# Retrieves the array of `runtime:CallStackElement` instances.
```

Comment:
Shall we add code snippet of this function

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/22534/commits/ba874a5eec5ea9827e2778a1fc28087e9160ffc6